Jenkinsfile

```diff
@@ -759,6 +759,69 @@ pipeline {
             }
         }
 
+
+        // Functional tests using BDD Stack
+        // See https://github.com/BCDevOps/BDDStack
+        stage('TEST - Smoke Tests') {
+            when {
+                expression { env.CHANGE_TARGET == 'master' }
+            }
+            steps {
+                script {
+                    _openshift(env.STAGE_NAME, TOOLS_PROJECT) {
+                        echo "Functional Testing"
+                        String baseURL = "https://${TEST_HOST}/gwells"
+                        podTemplate(
+                            label: "bddstack-${PR_NUM}",
+                            name: "bddstack-${PR_NUM}",
+                            serviceAccount: 'jenkins',
+                            cloud: 'openshift',
+                            containers: [
+                                containerTemplate(
+                                    name: 'jnlp',
+                                    image: 'docker-registry.default.svc:5000/moe-gwells-tools/bddstack:latest',
+                                    resourceRequestCpu: '800m',
+                                    resourceLimitCpu: '800m',
+                                    resourceRequestMemory: '4Gi',
+                                    resourceLimitMemory: '4Gi',
+                                    workingDir: '/home/jenkins',
+                                    command: '',
+                                    args: '${computer.jnlpmac} ${computer.name}',
+                                    envVars: [
+                                        envVar(key:'BASEURL', value: baseURL),
+                                        envVar(key:'GRADLE_USER_HOME', value: '/var/cache/artifacts/gradle')
+                                    ]
+                                )
+                            ],
+                            volumes: [
+                                persistentVolumeClaim(
+                                    mountPath: '/var/cache/artifacts',
+                                    claimName: 'cache',
+                                    readOnly: false
+                                )
+                            ]
+                        ) {
+                            node("bddstack-${PR_NUM}") {
+                                //the checkout is mandatory, otherwise functional tests would fail
+                                echo "checking out source"
+                                checkout scm
+                                dir('functional-tests') {
+                                    try {
+                                        sh './gradlew -DchromeHeadlessTest.single=WellDetails chromeHeadlessTest'
```

Comment:
I don't think this test exists any more?

> ./gradlew -DchromeHeadlessTest.single=WellDetails chromeHeadlessTest
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Execution failed for task ':chromeHeadlessTest'.
> > Could not find matching test for pattern: WellDetails


This might be a good alternative? (It does more, but it's all non-destructive)

`./gradlew -DchromeHeadlessTest.single=SearchSpecs chromeHeadlessTest`

Reply:
Also done!